Prototypes.md

```diff
@@ -0,0 +1,25 @@
+## Prototypes
```

Comment:
I think it's OK to just delete this text. It's not the focus of this repo. We won't delete the branches since we'll have them around for the historical record, but we don't need a top-level file explaining them.